lib/tasks/install_secrets.rake

```diff
@@ -0,0 +1,84 @@
+require 'aws-sdk-ssm'
```

Comment:
largely copied from open-search

Reply:
seems like it might be a good idea to pull the code out of the rake task into a generalized and shared spot, with unit tests?  e.g., in the aws ruby gem?